tests/sles4sap/sap_suse_cluster_connector.pm

```diff
@@ -29,7 +29,7 @@ sub exec_conn_cmd {
     assert_script_run("$args{binary} $cmd", timeout => $timeout);
     if ($args{log_file}) {
         my $output = script_output("cat $args{log_file}", proceed_on_failure => 1);
-        record_info("Command output", "$output");
+        record_info("Command output", "${output}");
```

Comment:
Change not needed.

Reply:
Curly braced variable is safely enclosed variable.

Reply:
It is for bash. Is it in perl too here?